frappe/contacts/doctype/contact/contact.js

```diff
@@ -32,11 +32,15 @@ frappe.ui.form.on("Contact", {
 			});
 		}
 		frm.set_query('link_doctype', "links", function() {
+			// company_description to include Company in the filtered options
+
+			let fieldtypes = ["HTML", "Text Editor"];
+			let fieldnames = ["contact_html", "company_description"];
```

Comment:
This assumes that `erpnext` app will always be installed alongside `frappe`. `frappe` shouldn't contain `erpnext` specific changes. This should be moved inside `erpnext` repo.

You can do this using `doctype_js` hook: https://frappeframework.com/docs/user/en/python-api/hooks#override-form-scripts